### DISCUSSION TITLE
Prime Rib (special drop like ham)

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#692] [Ideas]
Unlike the pig, the cow is noticeably lacking a special drop, and as such, there appears to be no reason to kill it with a knife.

I suggest "Prime Ribs" or something?

While youre at it: a "drumstick" drop from chickens too? Can be cooked into "fried chicken" then 3 crafted into a "bucket of chicken"

**Comment by vectorwing** [comment]
The main reason to slay other animals with a Knife is to guarantee their secondary drop: +1 Leather from cows and horses, +1 Feather from chickens etc.

Pigs provide a special drop because, otherwise, they are the least useful farm animal to herd. Ham makes them give both more meat and a source of bones/bone meal, as well as unlock a new feast.

Not sure about adding such a new drop to the Cow, given it's already the most useful animal to farm. Depends on potential feast ideas later.

**Reply by Aceplante** [reply]
Perhaps it could be much rarer then, but I think this would be a cool idea
